scout/adapter/mongo/case.py

```diff
@@ -102,24 +103,20 @@ def _set_similar_phenotype_query(
                 hpo_terms.append(term.get("phenotype_id"))
             similar_cases = self.cases_by_phenotype(hpo_terms, institute_id, case_obj["_id"])
         else:  # similar HPO terms
-            hpo_terms = list(query_term.replace(" ", "").split(","))
+            hpo_terms = list(query_value.replace(" ", "").split(","))
             similar_cases = self.cases_by_phenotype(hpo_terms, institute_id, None)
 
         if len(similar_cases) == 0:  # No cases similar to given phenotype
             query["_id"] = {"$in": []}  # No result should be returned by query
             return
 
         similar_case_ids = []
-        order = []
         for i in similar_cases:
             similar_case_ids.append(i[0])
-            order.append(i[1])
```

Comment:
This is weird and I don't know how we were thinking when we wrote the function like this. 

Testing locally: `similar_cases` looks like this:

<img width="178" alt="image" src="https://github.com/user-attachments/assets/5e617fc2-8d78-4d97-a266-1a576c1c5b17">

So `order` will be:  [1.0, 1.0, 1.0]

This value is returned by the function.